exercises/conversions/try_from_into.rs

```diff
@@ -19,9 +19,14 @@ struct Color {
 // You need to create an implementation for a tuple of three integers,
 // an array of three integers and a slice of integers.
 //
-// Note that the implementation for tuple and array will be checked at compile time,
+// Note that correct RGB color values must be integers in the 0..=255 range.
+//
+// Also note that the implementation for tuple and array will be checked at compile time,
 // but the slice implementation needs to check the slice length!
-// Also note that correct RGB color values must be integers in the 0..=255 range.
+// A little hint:
```

Comment:
This is tripping me up right now. I hadn't come across this syntax in any of my Rust research and I don't know how to get past it. I'll try this "little hint" now but I think Rustlings should have a dedicated rustling prior to this so we can learn how to do this properly.